openapi-cli/src/main/java/io/ballerina/openapi/cmd/utils/CodegenUtils.java

```diff
@@ -36,6 +40,24 @@ public static Path getImplPath(String pkg, Path srcPath) {
         return (pkg == null || pkg.isEmpty()) ? srcPath : srcPath.getParent();
     }
 
+    /**
+     * Resolves path to write generated implementation source files.
+     *
+     * @param licenseFilePath     path of the file which contains the content of license header
+     */
+    public static void setLicenseHeader(String licenseFilePath) throws IOException {
+        licenseHeader = "";
```

Comment:
why this variable re-init again?

Reply:
Since the methods in this class are static, to successfully run the test cases we have to clear the value here.

Reply:
Updated the implementation. Please review.